README.md

```diff
@@ -1,180 +1,144 @@
 ```
 
 We use the `--rm` option because the extracted maps are written to the host,
 thus the Docker container is no longer needed.
 
-Once the maps are extracted we can mount the maps from the host into a data-only
-container. The resulting container will be used by the `worldserver`.
-
-```sh
-$ docker create -it --name tc-maps -v ~/WoW/ServerData:/opt/trinitycore/maps trinitycore data
-```
+Once the maps are extracted to the volume, we can use it in the worldserver container.
 
-#### Advanced: Extracting maps into a Docker image
+### help
 
-It is possible to extract maps into a Docker image. This is useful if you
-plan on deploying to a remote server. Like before, assume the WoW client
-is located at `~/WoW/WoW3.3.5a`.
+Run the help command!
 
 ```sh
-$ docker run --name map-container -it -v ~/WoW/WoW3.3.5a:/opt/wow-client trinitycore extract-maps
+docker run --rm -it trinitycore help
 ```
 
-We do __not__ use the `--rm` option in this case because the maps are written
-into the Docker container's file system, not the host's. We also do not mount
-`~/WoW/ServerData` to `/opt/trinitycore/maps`. When the extraction is complete,
-the container will be stopped, but still exists. It can be found with
+# Worldserver and Authserver
+
+The worldserver and authserver both depend on a database connection. Docker's `--link` command effectively exposes the ports of a running container to a new container 
+via environment variables. Therefore, the [database container](db/README.md) must be running before attempting to start either the auth or world servers.
 
-```sh
-$ docker ps -a | grep map-container
-```
+It is assumed that the name of the running db container is `tc-dbserver`.
 
-Now the container can be committed into an image called `trinitycore-maps`.
+The worldserver depends on the extracted maps accessible at `/usr/local/trinitycore/data`. In the below examples, this is provided by the volume we have created 
+and populated above: TC-maps.
+ 
+NOTE: in the following commands, the `-i` is very important when running in daemon mode. Without it, the servers will not actually start or will not later be accessible 
+via interactive prompt using `docker exec`.
 
-```sh
-$ docker commit map-container trinitycore-maps
-```
+## Running the worldserver for the first time.
 
-After `map-container` is committed to an image, the container is not needed and
-can be safely deleted. The resulting image can be pushed to a Docker registry,
-and can be used to create data-only containers to use with `worldserver`.
+As the worldserver will ask if we want to create the database the first time (even with the worldserver.conf Updates section enabled), we will need to connect in interactive mode.
 
 ```sh
-$ docker run -it --name tc-maps -v /opt/trinitycore trinitycore-maps data
+docker run --name tc-worldserverinit --rm -it --link tc-dbserver:TCDB -p 8085:8085 -v TC-maps:/usr/local/trinitycore/data -v TC-config:/usr/local/trinitycore/etc trinitycore worldserver
```

Comment:
```suggestion
docker run --rm -it --link tc-dbserver:TCDB -p 8085:8085 -v TC-maps:/usr/local/trinitycore/data -v TC-config:/usr/local/trinitycore/etc trinitycore worldserver
```

I don't think the name is worth specifying here since it's just a temporary container!